From: Simon McVittie <smcv@debian.org>
Date: Mon, 7 Mar 2022 21:07:25 +0000
Subject: s390x-se-luks-gencpio: Use interoperable path for bash

On OSs that do not consistently merge /usr/bin with /bin, the path to
bash has traditionally been /bin/bash.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 src/libostree/s390x-se-luks-gencpio | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libostree/s390x-se-luks-gencpio b/src/libostree/s390x-se-luks-gencpio
index 729aff6..7c94a44 100755
--- a/src/libostree/s390x-se-luks-gencpio
+++ b/src/libostree/s390x-se-luks-gencpio
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 # This script creates new initramdisk with LUKS config within
 set -euo pipefail
 
